code/src/App.js

```diff
@@ -1,9 +1,53 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { HappyThought } from 'components/HappyThought.js'
```

Comment:
Maks gave an example last week that it's good to structure the imported dependencies with a blank space between the outer dependencies (like React) and the local dependencies (like Happyform) and the css-files. I think it's more important if you have a lot of them, but it can be good to keep in mind. 

So all outer dependencies goes together 
(then blank space ) 

all locals together 
(then blank space )

all css-files together.

Reply:
👍